apps/remote_control/lib/lexical/remote_control/code_intelligence/entity.ex

```diff
@@ -465,4 +477,21 @@ defmodule Lexical.RemoteControl.CodeIntelligence.Entity do
       _ -> nil
     end
   end
+
+  defp maybe_local_capture_func(analysis, position) do
+    with {:ok, %Zipper{node: {:/, _, [_, {:__block__, _, _}]}} = zipper} <-
+           Ast.zipper_at(analysis.document, position),
+         %Zipper{node: {:&, _, _}} <- Zipper.up(zipper) do
```

Comment:
Can we wrap this cryptic match in a function called `inside_capture?` ?
This is one of those things that'll make little sense in six months.

Reply:
Sure